cardano-testnet/src/Testnet/Process/Cli.hs

```diff
@@ -116,6 +121,16 @@ cliNodeKeyGen tmpDir vkey skey counter = do
     ]
   return (File vkPath, File skPath, File counterPath)
 
+-- | Call a command of the CLI that returns JSON and parse it
```

Comment:
It's not clear from where this JSON is retrieved. Most of CLI commands print human-readable text to stdout, and json is dumped only using `--out-file`. This may lead to some surprises when calling commands. Can you clarify this in this comment here?

I think this function should also be renamed to reflect that. I would also suggest sticking to `execCli*` naming convention (I needed to think for a moment what this new function does, and why it is different than `execCli*`).

What do you think about `execCliStdoutToJson` name for example?

Reply:
I'm taking this comment to the spinoff PR I did for this change: https://github.com/IntersectMBO/cardano-node/pull/5664